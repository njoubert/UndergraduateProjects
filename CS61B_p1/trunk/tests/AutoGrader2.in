John is the professor.
Jack is not the professor.
John lives in the green house.
Jack does not live in the green house.
The professor lives in the green house.
The churchwarden does not live in the green house.
John lives around here.
There is a blue house.
The churchwarden lives around here.
What do you know about the professor?
What do you know about Jack?
What do you know about the blue house?
Who lives in the green house?
What does the occupant of the blue house do?
What does Jack do?
Where does Jack live?
Where does the churchwarden live?

