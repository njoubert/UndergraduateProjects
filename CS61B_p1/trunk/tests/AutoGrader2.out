1. John is the professor.
2. Jack is not the professor.
3. John lives in the green house.
4. Jack does not live in the green house.
5. The professor lives in the green house.
6. The churchwarden does not live in the green house.
7. John lives around here.
8. There is a blue house.
9. The churchwarden lives around here.
Q: What do you know about the professor?
A: John is the professor and lives in the green house.
Q: What do you know about Jack?
A: Jack is the churchwarden and lives in the blue house.
Q: What do you know about the blue house?
A: Jack is the churchwarden and lives in the blue house.
Q: Who lives in the green house?
A: John lives in the green house.
Q: What does the occupant of the blue house do?
A: The churchwarden lives in the blue house.
Q: What does Jack do?
A: Jack is the churchwarden.
Q: Where does Jack live?
A: Jack lives in the blue house.
Q: Where does the churchwarden live?
A: The churchwarden lives in the blue house.
